python/analyzers/HNLTreeProducer.py

```diff
@@ -124,20 +188,44 @@ def process(self, event):
         '''
         self.readCollections(event.input)
         self.tree.reset()
-
+        self.event = event
```

Comment:
what is this?

Reply:
good you saw it, it's not needed anymore, was formerly used to implement something... i removed it